src/modules/auth/index.js

```diff
@@ -1,11 +1,15 @@
+import { lazy } from 'react';
 import Module from '../../abc/Module';
 
 import HeaderComponent from './header'
 import reducer from './reducer';
 import { types } from './actions'
-import { SeaCatAuthApi, GoogleOAuth2Api } from './api';
-import AccessControlScreen from './AccessControlScreen';
-import { locationReplace } from 'asab-webui';
+import { SeaCatAuthApi } from './api';
+import { locationReplace, componentLoader } from 'asab-webui';
+const AccessControlScreen = lazy(() => componentLoader(() => import('./AccessControlScreen')));
+
+import "./components/unauthorizedaccess.scss";
```

Comment:
@petrKavulok This import does not have to be here, since unauthorized access is not loaded within auth index module